src/deriv_api/SubscriptionManager.js

```diff
@@ -2,6 +2,7 @@ import {
     first, finalize, share,
 } from 'rxjs/operators';
 
+import streams_list                       from './StreamsList';
```

Comment:
for the import , in which case it need {} or not ?

Reply:
when we export a dict , and want to only import one key, we use {}.

Reply:
hi , where can i see which deriv shares this stream leads .thanks chylli .

Reply:
thanks i found it  !    39  - this.streams_list =[ 'balance' , etc ...]   
marvellous to see how it would be traded . as a learner at API coding orders ! 
thanks further info are appreciated . 
paul macques